CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing Guidelines
+
+Thanks for your interest in contributing to the `notebooks` repository.
+
+### Is this your first contribution?
+
+Please take a few minutes to read GitHub's guide on [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).
+It's a quick read, and it's a great way to introduce yourself to how things work behind the scenes in open-source projects.
+
+### We actively welcome your pull requests!
+
+If you want to update the documentation, [README.md](README.md) is the file you're looking for.
+
+Pull requests are the best way to propose changes to the notebooks repository:
+
+- Configure name and email in git
+- Fork the repo and create your branch from main.
+- Sign off your commit using the -s, --signoff option. Write a good commit message (see [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/))
+- If you've added code that should be tested, [add tests](https://github.com/openshift/release/blob/master/ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml).
+- Ensure the test suite passes.
+- Make sure your code lints.
+- Issue that pull request!
+
+### Some basic instructions to create a new notebook
+
+- Decide from which notebook you want to derive the new notebook
+- Create a proper filepath and naming to the corresponding folder
+- Add the minimum files you have to add:
+    - requirments.in with the additional packages
+    - Dockefile with proper instructions
+    - Kustomization objects to deploy the new notebook into an openshift cluster (Kustomization.yaml, service.yaml, statefulset.yaml)
+- Create instructions into Makefile, for example if you derive the new notebooks from minimal then the recipe should be like the following:
+    ```
+    # Your comment here
+    .PHONY: jupyter-${NOTEBOOK_NAME}-ubi8-python-3.8
+    jupyter-${NOTEBOOK_NAME}-ubi8-python-3.8: jupyter-minimal-ubi8-python-3.8
+	$(call pip_compile,bootstrap/ubi8-python-3.8,\
+		base/ubi8-python-3.8/requirements.in jupyter/minimal/ubi8-python-3.8/requirements.in jupyter/${NOTEBOOK_NAME}/ubi8-python-3.8/requirements.in,\
+		jupyter/${NOTEBOOK_NAME}/ubi8-python-3.8/requirements.txt)
+	$(call image,$@,jupyter/${NOTEBOOK_NAME}/ubi8-python-3.8,$<)
+    ```
+- Test the changes locally, by manually running the `$ make jupyter-${NOTEBOOK_NAME}-ubi8-python-3.8` from the terminal.
```

Comment:
Shouldn't this just be `make ${NOTEBOOK_NAME} https://github.com/opendatahub-io/notebooks#building ?

Reply:
Yes and no, yes in order to keep consistency with the readme instructions, and no because actually when you run the `make` command should be followed by the name of the recipe